hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/ClientInvocationServiceImpl.java

```diff
@@ -54,7 +53,7 @@
     private static final HazelcastProperty IDLE_STRATEGY
             = new HazelcastProperty("hazelcast.client.responsequeue.idlestrategy", "block");
 
-    private static final int WAIT_TIME_FOR_PACKETS_TO_BE_CONSUMED_THRESHOLD = 5000;
+    private static final int CLEAN_RESOURCES_INTERVAL_MILLIS = 100;
```

Comment:
I think, this is too internal to expose. It can be changed/deleted. If we expose, we need to keep it.